src/Me/Modals/MentorshipRequest.js

```diff
@@ -0,0 +1,187 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Modal } from './Modal';
+import FormField from '../components/FormField';
+import Textarea from '../components/Textarea';
+import { desktop } from '../styles/shared/devices';
+import { toast } from 'react-toastify';
+import { sendMentorshipRequest } from '../../api';
+import messages from '../../messages';
+import ImageSrc from '../../assets/mentorshipRequestSuccess.svg';
+
+const MentorshipRequestDetails = styled.div`
+  margin: 0 auto;
+  width: 100%;
+  max-width: 52rem;
+`;
+
+const MentorshipRequestDetailsForm = styled.form`
+  display: flex;
+  width: 100%;
+`;
+
+const SuccessMentorshipRequest = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const FormFields = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: stretch;
+  padding: 0 5px;
+  width: 100%;
+`;
+
+const ExtendedFormField = styled(FormField)`
+  flex: 1 1 100%;
+  /* max-width: 355px; */
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+
+  & label {
+    color: #4f4f4f;
+  }
+
+  width: 100%;
+  min-width: 15rem;
+
+  @media ${desktop} {
+    flex: ${props => (props.customFormField ? '1 1 100%' : '1 1 40%')};
+    max-width: ${props => (props.customFormField ? 'unset' : '355px')};
+  }
+`;
+
+const model = {
+  myBackground: {
+    label: 'My Background',
+    type: 'longtext',
+    defaultValue: '',
+    placeholder: 'Tell the mentor about yourself.',
+  },
+  myExpectations: {
+    label: 'My Expectations',
+    type: 'longtext',
+    defaultValue: '',
+    placeholder: 'What do you expect from this mentorship?',
+  },
+  message: {
+    label: 'Message',
+    type: 'longtext',
+    defaultValue: '',
+    validate: value => !!value,
+    required: true,
+    placeholder: 'Anything else you want to say?',
+  },
+};
+
+const MentorshipRequest = ({ mentor, closeModal }) => {
+  const [confirmed, setConfirmed] = useState(false);
+  const [mentorshipRequestDetails, setMentorshipRequestDetails] = useState({});
+
+  // text fields onChange function
+  const handleInputChange = (fieldName, value) => {
+    setMentorshipRequestDetails({
+      ...mentorshipRequestDetails,
+      [fieldName]: value,
+    });
+  };
+
+  const formField = (fieldName, config) => {
```

Comment:
Can we get the values from [localStorage](https://github.com/Coding-Coach/find-a-mentor/pull/747/files#diff-9164e2522095f9ab4a441b3219f191a5845573c92a41ccc142aee15449c7fb48R240) if exists?

Reply:
This is solved, right?

Reply:
yes